README.md

```diff
@@ -1,37 +1,43 @@
+ At a minimum, the​ ​app​ should ​allow​ ​us​ ​to​ ​access​ ​a​ ​lobby and​ ​create​ ​a​ ​game​ ​where​ ​you​ ​invite others​. If you want to and have enough time you can add bonus features: ​players’​ ​details view, ​rearranging ​teams​ or,​ ​after​ ​a​ ​match,​ ​add​ing and storing ​some​ ​stats.
 
- We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ ​If​ ​you​ ​don’t​ ​manage​ ​to​ ​implement everything,​ ​no​ ​worries​ ​-​ ​we’d​ ​be​ ​very​ ​pleasantly​ ​surprised​ ​if​ ​you​ ​did.
+ We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ Don't worry if ​you​ ​don’t​ ​manage​ ​to​ ​implement a gazillion of features or breathtaking design -​ focus on the create game and lobby functionalities. We'll appreciate clean elegant code, a thoughtful solution and a nice project structure!
  
  # The​ ​functionality
  - Lobby​ ​(with​ ​a​ ​list​ ​of​ ​players​ ​and​ ​some​ ​information) 
- - Detailed​ ​player​ ​view
  - Create​ ​a​ ​game 
     - Invite​ ​players
-    - Rearrange​ ​teams​ ​by​ ​reassigning​ ​players 
-    - Validation​ ​(teams​ ​can​ ​only​ ​consist​ ​of​ ​4​ ​or​ ​2​ ​players)
+
+
+## Nice to haves
 - Mark​ ​which​ ​team​ ​won 
-## Extra​ ​brownie​ ​points​ ​ideas 
+- Rearrange​ ​teams​ ​by​ ​reassigning​ ​players 
+- Detailed​ ​player​ ​view
 - Add​ ​some​ ​stats​ ​to​ ​each​ ​player
 - Save​ ​stats
- - Login​ ​(can​ ​be​ ​mocked)
- - Visually​ ​pleasing​ ​design​ ​and​ ​interactions
+- Login​ ​(can​ ​be​ ​mocked)
+- Visually​ ​pleasing​ ​design​ ​and​ ​interactions
  
- # Look​ ​and​ ​feel 
- We​ ​provide​ ​no​ ​design,​ ​bar​ ​a​ ​few​ ​(deliberately)​ ​ugly​ ​wireframes.​ ​Any​ ​aesthetic​ ​decisions​ ​are​ ​up to​ ​you;​ ​obviously,​ ​any​ ​effort​ ​put​ ​in​ ​this​ ​area​ ​will​ ​be​ ​appreciated​ ​and​ ​taken​ ​into​ ​account. The​ ​only​ ​requirement​ ​is:​ ​​it​ ​has​ ​to​ ​be​ ​responsive​ ​​and​ ​work​ ​well​ ​on​ ​a​ ​variety​ ​of​ ​platforms, browsers​ ​(the​ ​lowest​ ​we​ ​support​ ​is​ ​IE11),​ ​devices. 
+ ## Look​ ​and​ ​feel 
+ We​ ​provide​ ​no​ ​design so any​ ​aesthetic​ ​decisions​ ​are​ ​up to​ ​you. The​ ​only​ ​requirement​ ​is:​ ​​it​ ​has​ ​to​ ​be​ ​responsive​ ​​and​ ​work​ ​well​ ​on​ ​a​ ​variety​ ​of​ ​platforms, browsers​ ​(the​ ​lowest​ ​we​ ​support​ ​is​ ​IE11),​ ​devices. 
 
-#Expected​ ​minimum​ ​tech​ ​stack
+## Expected​ ​minimum​ ​tech​ ​stack
 - JS​ ​(ES6),​ ​HTML,​ ​CSS 
 - A​ ​framework​ ​(preferably​ ​Angular) 
-- Npm 
-- A​ ​css​ ​preprocessor Assets​ ​provided 
+- npm 
+
+But in general we want you to take advantage of tools you know and like to use.
+
+## Assets​ ​provided 
 - Players’​ ​photos 
 - ​JSON​ ​with​ ​players’​ ​details 
-- Basic​ ​wireframes Expected​ ​outcome 
+- Basic​ ​wireframes 
+
+## Expected​ ​outcome 
 - A​ ​repository​ ​with​ ​project​ ​structure​ ​clearly​ ​visible
```

Comment:
we mentioned 'structure' twice. I would prefer not to empesis this as I would like to know what they will do without us giving a hint. now they know it's important to us.

Reply:
Sure.